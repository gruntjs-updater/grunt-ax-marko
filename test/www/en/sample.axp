
	
	
<%@ page contentType="text/html;charset=UTF-8" language="java" %>
<%@ taglib prefix="sec" uri="http://www.springframework.org/security/tags" %>
<%@ taglib prefix="spring" uri="http://www.springframework.org/tags" %>
<% %>
<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<title></title>
</head>
<body>

	<h1>
        My Page
    </h1>

	

		Hello World!

		<ul>
			<li style="color: red">
				red
			</li><li style="color: green">
				green
			</li><li style="color: blue">
				blue
			</li>
		</ul>

		Hello World!

	

</body>
</html>