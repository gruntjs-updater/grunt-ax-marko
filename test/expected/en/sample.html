
	
	

<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<title></title>
</head>
<body>

	<h1>
        My Page
    </h1>

	

		Hello World!

		<ul>
			<li style="color: red">
				red
			</li><li style="color: green">
				green
			</li><li style="color: blue">
				blue
			</li>
		</ul>

		Hello World!

	

</body>
</html>